### PR TITLE
tighten_type_of_initializing_formals.dart typo

### PR DESCRIPTION
**Pull request by FMorschel**
tighten_type_of_initializing_formals typo fixed at line 40

### CODE REVIEW

**Review by bwilkerson — APPROVED** [review]

**Comment by pq** [comment]
Thanks for catching this!

Could you add an attribution for yourself to [`AUTHORS`](https://github.com/dart-lang/linter/blob/master/AUTHORS)?

**Comment by FMorschel** [comment]
> Thanks for catching this!
> 
> Could you add an attribution for yourself to [`AUTHORS`](https://github.com/dart-lang/linter/blob/master/AUTHORS)?

Done

File: AUTHORS [hunk 1]
```diff
@@ -41,3 +41,4 @@ Minh Dao <asiancorporator@yahoo.de>
 Sai Sandeep Mutyala <rageofepicfury@gmail.com>
 Kristen McWilliam <kmcwilliampublic@gmail.com>
 Paul Köhler <kontakt@paul-koehler.dev>
+Felipe Morschel <felipe.morschel@gmai.com>
```
Comment (pq) [inline AUTHORS:44]: Should this read `@gmail.com`?

Comment (FMorschel) [reply AUTHORS]: Yes, already updated it, thanks

**Review by pq — APPROVED** [review]

**Pull request merged**